pnet_datalink/src/lib.rs

```diff
@@ -60,7 +61,8 @@ mod backend;
 pub mod netmap;
 
 #[cfg(feature = "pcap")]
-pub mod pcap;
+#[path = "pcap.rs"]
+pub mod backend;
```

Comment:
I don't think `pcap` should ever be the default backend - the default backend should be libpnet's native backend on the platform.

There is an argument for having it as a fallback backend for platforms we don't have native support for - I don't know that that's a good idea though - what if we add a native backend in the future? That potentially breaks code.